distribution/rpm/src/main/packaging/init.d/elasticsearch

```diff
@@ -35,8 +35,6 @@ fi
 ES_HOME="/usr/share/elasticsearch"
 MAX_OPEN_FILES=65536
 MAX_MAP_COUNT=262144
-LOG_DIR="/var/log/elasticsearch"
```

Comment:
That would be the only place, these environment variables were only ever used in the service files and were never exported from them. Let me do this as a follow up.